.travis.yml

```diff
@@ -62,8 +62,8 @@ install:
 
 before_script:
   - echo 'CREATE DATABASE phanbook CHARSET=utf8 COLLATE=utf8_unicode_ci' | mysql -u root
-  - cat schema/phanbook.sql | mysql -uroot phanbook
-  - cat schema/examples.sql | mysql -uroot phanbook
+  - cat schema/V1.0.0__Phanbook.sql | mysql -uroot phanbook
+  - cat schema/V1.0.1__AddExamples.sql | mysql -uroot phanbook
```

Comment:
@duythien Should we update docs regarding migrations?

Reply:
Yes we should do it, I recommend use tool https://flywaydb.org/